### PR TITLE
Update Docs for PowerFlex issue 1210

### PR DESCRIPTION
**Pull request by suryagupta4**
# Description
This PR includes documentation changes for the PowerFlex issue 1210.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| [#1210](https://github.com/dell/csm/issues/1210) |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by adarsh-dell — APPROVED** [review]
LGTM

**Review by AkshaySainiDell — APPROVED** [review]
lgtm

File: content/docs/deployment/helm/drivers/installation/powerflex.md [hunk 1]
```diff
@@ -213,10 +213,11 @@ Use the below command to replace or update the secret:
 | skipCertificateValidation | A boolean that enables/disables certificate validation of the csm-authorization proxy server. | No | true |
 
 
-10. Install the driver using `csi-install.sh` bash script by running `cd dell-csi-helm-installer && ./csi-install.sh --namespace vxflexos --values myvalues.yaml`. You may modify the release name with the `--release` arg. If arg is not provided, release will be named `vxflexos` by default. 
+10. Install the driver using `csi-install.sh` bash script by running `cd dell-csi-helm-installer && ./csi-install.sh --namespace vxflexos --values myvalues.yaml --helm-charts-version csi-vxflexos-2.10.1`. You may modify the release name with the `--release` arg. If arg is not provided, release will be named `vxflexos` by default.
```
Comment (gallacher) [inline content/docs/deployment/helm/drivers/installation/powerflex.md:216]: Why is this here? Look at previous comment.

File: content/docs/deployment/helm/drivers/installation/powerflex.md [hunk 2]
```diff
@@ -213,10 +213,11 @@ Use the below command to replace or update the secret:
...
+10. Install the driver using `csi-install.sh` bash script by running `cd dell-csi-helm-installer && ./csi-install.sh --namespace vxflexos --values myvalues.yaml --helm-charts-version csi-vxflexos-2.10.1`. You may modify the release name with the `--release` arg. If arg is not provided, release will be named `vxflexos` by default. 
 Alternatively, to do a helm install solely with Helm charts (without shell scripts), refer to `helm/README.md`.
 
  *NOTE:*
+- If you do not specify the `--helm-charts-version` flag, by default the `csi-install.sh` script will clone the version of the helm chart that is specified in the driver's [csi-install.sh](https://github.com/dell/csi-powerflex/blob/main/dell-csi-helm-installer/csi-install.sh#L24) file. If you wish to install the driver using a different version of the helm chart, you need to include this flag. Also, remember to delete the `helm-charts` repository present in the `csi-powerflex` directory if it was cloned before.
```
Comment (gallacher) [inline content/docs/deployment/helm/drivers/installation/powerflex.md:220]: Similar to comments above

Comment (adarsh-dell) [reply content/docs/deployment/helm/drivers/installation/powerflex.md:220]: Hi @gallacher Thanks for the review.
Since we are not releasing the driver i.e. v2.10.1 so customer will clone/checkout the v2.10.0 branch that is the latest release of driver and will try to run the install script and by default it will clone the same version of helm chart that is matching with the driver version so here in this case we need to pass this flag to override the default value.
Earlier we have never mentioned this flag before so in case if customer is trying out with the older command then they will end up with installing the driver with older version of helm-chart that has the bug.
That's why we thought of introducing this Note point here to avoid any confusion.

Comment (suryagupta4) [reply content/docs/deployment/helm/drivers/installation/powerflex.md:220]: Hi @gallacher, adding to Adarsh's comment, during driver installation, the helm-charts repository is cloned with a specified branch (Ref: https://github.com/dell/csi-powerflex/blob/main/dell-csi-helm-installer/csi-install.sh#L384).
Since we are not releasing the driver, the branch `csi-vxflexos-2.10.0` will be cloned by default which is defined in the `csi-install.sh` script here https://github.com/dell/csi-powerflex/blob/main/dell-csi-helm-installer/csi-install.sh#L24 if we don't specify that flag explicitly.

Comment (gallacher) [reply content/docs/deployment/helm/drivers/installation/powerflex.md:220]: @adarsh-dell / @suryagupta4 - is this acceptable from a customer perspective? Please work with @sharmilarama and team to ensure we are aligned properly.

Comment (adarsh-dell) [reply content/docs/deployment/helm/drivers/installation/powerflex.md:220]: If that option is available, then there shouldn't be an issue with utilizing it, as this specific flag was introduced precisely to handle such scenarios.

File: content/docs/csidriver/release/powerflex.md [hunk 3]
```diff
@@ -43,3 +43,4 @@ A CSI ephemeral pod may not get created in OpenShift 4.13 and fail with the erro
 ### Note:
 
 - Support for Kubernetes alpha features like Volume Health Monitoring and RWOP (ReadWriteOncePod) access mode will not be available in Openshift environment as Openshift doesn't support enabling of alpha features for Production Grade clusters.
+- For fixing [#1210 - [BUG]: Helm deployment of PowerFlex driver is failing](https://github.com/dell/csm/issues/1210), a new helm-chart has been released. In order to install this helm chart, we need to pass the flag `--helm-charts-version` during helm installation and flag `-v` during offline bundle installation with value `csi-vxflexos-2.10.1`.
```
[the remark this answers is not in the document]
Comment (shanmydell) [reply content/docs/csidriver/release/powerflex.md:46]: @gallacher : Changes are needed only in helm-charts, we are not releasing the diver.

**Review by shanmydell — APPROVED** [review]
